code/game/machinery/adv_med.dm

```diff
@@ -209,7 +213,7 @@
 			to_chat(usr, "\icon[src]<span class='warning'>The body scanner cannot scan that lifeform.</span>")
 			return
 		var/obj/item/weapon/paper/R = new(src.loc)
-		R.name = "Body scan report"
+		R.name = "[occupant.real_name] scan report"
```

Comment:
Revert this.

Reply:
By revert this do you mean change it to
```suggestion
		R.name = "[occupant.name] scan report"
```
or to revert it to
```suggestion
		R.name = "Body scan report"
```
asking this bc you wanted the normal name in the comment below

Reply:
Go with get_visible_name, would be a body scanner QoL (the reason of your PR) so that it can be differentiated when ironhammer mass-scan people